src/components/AnnotationPopup/AnnotationPopup.js

```diff
@@ -81,7 +97,6 @@ class AnnotationPopup extends React.PureComponent {
         canModify: core.canModify(annotation)
       });
     } else {
-      this.props.closeElement('annotationPopup');
```

Comment:
We actually need this line, otherwise the annotation popup remains open when the annotation is deselected from the notes panel

Reply:
I put back the line of code that was removed. 

https://github.com/PDFTron/webviewer-ui/pull/68/commits/8bb854e8129f588ce6bdf58912bc5b899cd41175